<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecognitionDlg</class>
 <widget class="QDialog" name="RecognitionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="my_qlabel" name="lblRecogimg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px  solid #000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>350</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px  solid #000;</string>
   </property>
   <property name="title">
    <string>Recognition Shift Order</string>
   </property>
   <widget class="QLineEdit" name="leRecogSeq">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>Order</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbRecogrepeattime">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>30</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>976</x>
     <y>310</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 0px;</string>
   </property>
   <property name="text">
    <string>Recognition Mode</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbBRROI">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>120</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px  solid #000;</string>
   </property>
   <property name="title">
    <string>BRROI</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="leBRROISX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leBRROISY">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leBRROIWIDTH">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leBRROIHEIGHT">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>SX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>SY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>WIDTH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>HEIGHT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbBRROI">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="cbRecogMode">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>310</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ALPR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>iLPR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ALPR&amp;iLPR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>iLPR&amp;ALPR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dual ALPR&amp;iLPR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dual iLPR&amp;ALPR</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>570</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save /Apply</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>430</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px  solid #000;</string>
   </property>
   <property name="title">
    <string>Save Raw Image</string>
   </property>
   <widget class="QLineEdit" name="leRawsavetime">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>Number of save</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRawsaverecog">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>Recog</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRawsaveall">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRawsavemiss">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>No Recog</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbPlateROI">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>10</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px  solid #000;</string>
   </property>
   <property name="title">
    <string>PlateROI</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="lePlateROISX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePlateROISY">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePlateROIWIDTH">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePlateROIHEIGHT">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>SX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>SY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>WIDTH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>HEIGHT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPlateROI">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>230</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px  solid #000;</string>
   </property>
   <property name="title">
    <string>Vehicle position</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="rbVPLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbVPCenter">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>Center</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbVPRight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="chkDisplay">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkSaveLog">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>540</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Log</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
